anchor-reference-server/src/main/java/org/stellar/anchor/reference/AnchorEventConsumerService.java

```diff
@@ -0,0 +1,143 @@
+    consumer.subscribe(
+        Arrays.asList(
+            "ap_event_quote_created",
+            "ap_event_transaction_created",
+            "ap_event_transaction_payment_received")); // TODO make this configurable
+    this.consumer = consumer;
+
+    while (!shutdown) {
+      try {
+        ConsumerRecords<String, AnchorEvent> consumerRecords =
+            consumer.poll(Duration.ofSeconds(10));
+        System.out.println(
+            "Anchor Reference Server - Messages received - " + consumerRecords.count());
+        consumerRecords.forEach(
+            record -> {
+              for (Header header : record.headers()) {
+                if (Objects.equals(header.key(), "type")) {
+                  String eventType = new String(header.value(), StandardCharsets.UTF_8);
+                  switch (eventType) {
+                    case "quote_created":
+                      Log.debug("anchor_platform_event - quote_created " + record.value());
+                      break;
+                    case "transaction_created":
+                      Log.debug(
+                          "anchor_platform_event - " + "transaction_created " + record.value());
+                      break;
+                    case "transaction_payment_received":
+                      Log.debug(
+                          "anchor_platform_event - "
+                              + "transaction_payment_received "
+                              + record.value());
+                      if (record.value() instanceof TransactionEvent) {
+                        TransactionEvent transactionEvent = (TransactionEvent) record.value();
+                        PatchTransactionsRequest txnRequest =
+                            PatchTransactionsRequest.builder()
+                                .records(
+                                    List.of(
+                                        PatchTransactionRequest.builder()
+                                            .id(transactionEvent.getId())
+                                            .status(TransactionEvent.Status.COMPLETED.status)
+                                            .amountFee(
+                                                new Amount(
+                                                    transactionEvent.getAmountFee().getAmount(),
+                                                    transactionEvent.getAmountFee().getAsset()))
+                                            .amountOut(
+                                                new Amount(
+                                                    transactionEvent.getAmountOut().getAmount(),
+                                                    transactionEvent.getAmountOut().getAsset()))
+                                            .build()))
+                                .build();
+                        PlatformApiClient platformClient =
+                            new PlatformApiClient("http://localhost:8080");
```

Comment:
You're mocking the transaction completion, right? Can you add a comment making it clear?